guides/dev/issues_tracker/README.md

```diff
@@ -26,14 +29,40 @@ Please provide the following when reporting new issues. The more information you
 When filing an issue, please bear these important things in mind:
 
  * **Submit separate reports for each issue.** Do not include more than one issue in the same report.
- * The "Priority", "Milestone", "Version", "Assign to" and "Keywords" fields are reserved for the core development team.
  * Make sure **you are using the latest CKEditor version**. Maybe your issue has already been fixed.
- * For a detailed guide on how to properly report a new issue go to the [Ticket Specs page](https://dev.ckeditor.com/wiki/TicketSpecs).
+ * Follow the instructions in the  **issue template** when reporting an issue.
 
 ## Reporting an Issue
 
 Before clicking the following link, make sure you have read the instructions above!
 
-If you are not a registered user of our tracker, take a minute to [create a new account in the CKEditor Developer Center](https://dev.ckeditor.com/register). Only authenticated users can create tickets.
+Only registered GitHub users may create issues, if you are not registered, take a minute to [create a GitHub account](https://github.com/join?source=header-repo).
+
+This is it. Once you log in, you will be ready to [report a CKEditor issue](https://github.com/ckeditor/ckeditor-dev/issues/new).
+
+## Issue State
```

Comment:
How about making simply a section `### Labels` ? Which would be semantically a child of "Reporting an Issue". And simplify introduction paragraph to information like "we use following labels (...)" (not necessarily that, but the message should be simply that).